miniz.c

```diff
@@ -1,3 +1,6 @@
+/*
+SPDX-License-Identifier: MIT
```

Comment:
The version of minz used here is licensed under the “unlicense” not MIT.

Rather than modifying the upstream sources the tags can be added to a file matching the name with “.license” appended, for example, minz.h.license.